zap/src/test/resources/org/zaproxy/zap/spider/parser/htmlform/FormWithFormactionButtons.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Form With Button Using Formaction - Spider HTML Form Parser</title>
```

Comment:
Should have examples with:
 - Buttons with `form` attributes (in and outside forms);
 - Buttons with `formaction` but of `type` `reset` and `button`. Also, good to have some with explicit `submit` type and with invalid values.

Reply:
Should have example containing invalid value and at least one button inside a form that has different `form` attribute.